CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+ComplianceAsCode Contribution Guidelines
+===
+
+We aim to create a welcoming environment for all contributors regardless of their age, gender, experience, nationality, or employer. However, to foster smooth collaboration we are asking contributors to adhere to the following rules.
+
+Issues and Pull Requests
+---
+
+Pull requests can’t be merged by the author, a review from another person is needed. If anybody raises material objections to the pull request, those objections should be addressed before the PR merge.
```

Comment:
self-merging should be allowed for typos. This is okay to enhance & allow.

Reply:
```suggestion
- Two-party review is needed for merging pull requests. This second set of eyes helps review the code for functionality and style, and helps create a shared understanding of why certain things are merged (or not!).
```

Reply:
I disagree with the idea to allow self-merging. You can always find somebody to do a quick review, and typos typically aren't urgent.

Related to @shawndwells suggestion, your formulation suggests that approval of a second party is stronger than material objections. The question of merging a pull request that is believed to be fundamentally flawed shouldn't be a question of two people who want to merge it against one who doesn't - those in favor of the merge should address such objection to show that they understand it.

Reply:
While typos aren't urgent and you disagree, many projects do it for typos. I reduces T&E which is important and needed.

Reply:
I'm on board with simple typo fixes. I could be frustrating and embarrassing to ask somebody to fix your typo. I would be on board even with small text modifications, but only *if they don't change* the meaning.